src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -11,43 +20,32 @@
 
 public class ArmSubsystem extends SubsystemBase implements Loggable {
     public enum State {
-        Pickup, Rest, PlacingA, PlacingB, PlacingC, PlacingD, PlacingE, Transit // Possible System States
-    }
-
-    private final double elbowPosToDegrees = 599.9;
-    private final double shoulderPosToDegrees = 342.8;
-    private final double stateTolerance = 2;
-
-    // System motor controllers
-    private final CANSparkMax m_shoulder = new CANSparkMax(ArmConstants.SHOULDER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_shoulderFollower = new CANSparkMax(ArmConstants.SHOULDER_FOLLOWER_ID, MotorType.kBrushless);
-    private final CANSparkMax m_elbow = new CANSparkMax(ArmConstants.ELBOW_ID, MotorType.kBrushless);
-
-    // System State - init in rest state
-    private State m_state_desired = State.Rest;
-    private State m_state_actual = State.Rest;
+        Pickup(0, 0), Rest(0, 0), PlacingA(Math.PI / 4, 0), PlacingB(0, 0), PlacingC(0,
+                Math.PI / 2), PlacingD(Math.PI / 2, Math.PI / 2), PlacingE(0, 0);

```

Comment:
Positions should be based off the motor controller values so that we can tune based on the rev client and directly import those numbers.

Reply:
Doesn't it make more sense to use rad and grab values from the CAD? Plus, we can always convert if necessary.

Reply:
No because there is chain and motor slop. So you'll never have it perfect like cad does there will always be a margin for error.

So if the CAD isn't reliable enough because of this slop why would we take the motor values and go through the step of converting to and from when we need to compare it to the values being seen on by the robot? It's extra steps and complexity for not enough benefit.

Reply:
Right, but if we, say, change the gearing, the position tuning will *all* be invalid. If we store angles and then change the gear ratio later on, the angles will be the same. The position numbers also represent nothing physical - it's very difficult to measure the difference that `1` position makes, but knowing that you have a `1.2 * PI` angle is easily measurable. We can also store these in degrees if you'd like, but I'd rather not. Up to you though, if you want we can still go for raw rotations.

Reply:
We're not changing the ratios. If we do then we will simply repeat the process to find the positions. It takes about two minutes.

Reply:
Ok.